_data/adipurnamk/adipurnamk.md

```diff
@@ -0,0 +1,7 @@
+---
+name: adipurnamk 
+institution: freelance 
+profile_pic: adipurnamk.png 
+quote: "I'm curious", Eru Chitanda
```

Comment:
Remove double quotes or change them to single quotes.

Reply:
oh, i see, thanks Mohammed
it's done